dts/riscv/it8xxx2.dtsi

```diff
@@ -292,6 +292,7 @@
 			ngpios = <8>;
 			label = "GPIO_A";
 			gpio-controller;
+			index = <0x0>;
```

Comment:
remove all of the
~~~C
                index=<X>
~~~

Reply:
Done.